composer.lock

```diff
@@ -23054,5 +23054,5 @@
         "php": ">=7.4"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
Not sure why this is downgrading for a courtyard update so maybe best to keep at 2.3.0?